docs/docs/writing-hello-world.md

```diff
@@ -1,45 +1,47 @@
 # Writing the sample enclave
 
 ## Prerequisites
-This tutorial assumes that you have completed the [Running your first enclave](running-hello-world.md) tutorial.
+* Complete the [Running your first enclave](running-hello-world.md) tutorial.
 
 ## Introduction
 
-In this tutorial, we will cover the following:
+Use this tutorial to:
 
-* Setting up a new Conclave project using [Conclave Init](conclave-init.md).
-* Modifying the files to create an implementation of the Conclave *sample app* which we explored in the
+* Set up a new Conclave project using [Conclave Init](conclave-init.md).
+* Modify the files to create an implementation of the Conclave *sample app* which we explored in the
   [previous tutorial](running-hello-world.md).
 
-We will be leveraging Conclave's built-in web host to write our Conclave app. To learn more about this, see
-[Conclave web host](conclave-web-host.md). You may want to implement your own host if you need custom functionality or
-you want to incorporate it into an existing server application of some kind. To learn more about that, see
-[Writing your own enclave host](writing-your-own-enclave-host.md).
+This tutorial uses [Conclave web host](conclave-web-host.md) to write a Conclave app.
```

Comment:
Do we even need this sentence? the web host is already mentioned when further down where it's relevant. No need to talk about something the user will have no idea about.